src/world.rs

```diff
@@ -941,7 +941,7 @@ impl<T: Send + Sync + 'static> Component for T {}
 /// This is just a convenient shorthand for `Send + Sync + Clone + 'static`, and never needs to be
 /// implemented manually.
 pub trait ComponentClone: Send + Sync + Clone + 'static {}
-impl<T: Send + Sync + Clone + 'static> ComponentClone for T {}
+impl<T: Component + Clone> ComponentClone for T {}
```

Comment:
What is the benefit of this trait compared to using `Component + Clone` directly?

Reply:
No benifit. Didn't think of using the alias

Reply:
In that case, let's not complicate the public API unnecessarily with an additional trait.